### ISSUE TITLE
Error for Pending test on the server

### ISSUE THREAD

**Issue by loirit** [practicalmeteor/meteor-mocha#7]
Attempting to create a [pending test](http://mochajs.org/#pending-tests) that on the server causes the following error to be thrown:

```
TypeError: Cannot call method 'toString' of undefined
    at Function.boundWrappedFunction.toString (packages/practicalmeteor:mocha-core/server.js:93:1)
    at MeteorPublishReporter.practical.mocha.MeteorPublishReporter.MeteorPublishReporter.cleanTest (packages/practicalmeteor:mocha/meteor/src/server/MeteorPublishReporter.coffee:144:22)
    at Runner.<anonymous> (packages/practicalmeteor:mocha/meteor/src/server/MeteorPublishReporter.coffee:78:27)
    at Runner.emit (events.js:117:20)
    [...]
```

Pending tests work just fine on the client.

**Comment by jsep** [comment]
Yes its a bug, you can use `it.skip('test', function(){})` until we fix this issue. Thanks!

**Comment by rbabayoff** [comment]
Fixed in v2.1.0_5. Let us know if it works.

**Comment by loirit** [comment]
It's working correctly for me now.
